lib/livebook_web/live/settings_live/autosave_path_component.ex

```diff
@@ -0,0 +1,36 @@
+defmodule LivebookWeb.SettingsLive.AutosavePathComponent do
+  use LivebookWeb, :live_component
+
+  @impl true
+  def render(assigns) do
+    ~H"""
+    <div>
+      <div class="flex items-center justify-between border border-gray-200 rounded-lg p-4">
+        <div class="h-full h-52">
+          <.live_component module={LivebookWeb.FileSelectComponent}
```

Comment:
We should have a "Change" button for opening the file select, so that we don't take this much space all the time. We have similar behaviour when saving a notebook :)